src/audio/crossover/crossover_generic.c

```diff
@@ -225,36 +225,29 @@ static void crossover_s32_default(struct comp_data *cd,
 				  int32_t num_sinks,
 				  uint32_t frames)
 {
-	/* Array to hold active sink streams; initialized to null */
```

Comment:
all this commit does is delete the comments you added, please check your commits to make sure they make sense before pushing them.